helper_methods.rb

```diff
@@ -292,3 +292,25 @@ def add_revision_statement(uri, description)
   post = @client.post(uri, resource.to_json)
   puts post.body
 end
+
+def get_all_resource_uris_for_institution()
+  #run through all repositories
+  resources_endpoints = []
+  repos_all = (3..12).to_a
```

Comment:
3..12 is fairly magical.  Are these ids for the repos?  Why start at 3?  A comment might make this easier to remember for you and other to understand.

Reply:
That's an ASpace thing: 1 and 2 are reserved for Lyrasis/administrative use. 3-12 are the Princeton repo id's.

Reply:
I've added comments in the helper methods